benchmarks/allocator_burn_test.cpp

```diff
@@ -0,0 +1,41 @@
+//////////////////////////////////////////////////////////////////////////////
+// Copyright (c) 2018, Lawrence Livermore National Security, LLC.
+// Produced at the Lawrence Livermore National Laboratory
+//
+// Created by David Beckingsale, david@llnl.gov
+// LLNL-CODE-747640
+//
+// All rights reserved.
+//
+// This file is part of Umpire.
+//
+// For details, see https://github.com/LLNL/Umpire
+// Please also see the LICENSE file for MIT license.
+//////////////////////////////////////////////////////////////////////////////
+#include <chrono>
+#include <functional>
+#include <iostream>
+#include <random>
+
+#include "umpire/config.hpp"
+
+#include "umpire/Allocator.hpp"
+#include "umpire/ResourceManager.hpp"
+
```

Comment:
I'm not sure I understand the significance of this burn test within the context of a benchmark?  Is it being timed somewhere?  Perhaps a comment would be helpful to describe how it is used and what the expected outputs are?

Reply:
@jdahm, can you answer my question?  What is the intention of a "burn test" and how does it relate to a benchmark other than being in the benchmarks directory?

Reply:
This benchmark will be removed before merging.

Reply:
Thanks, I will leave this comment unresolved so that we are reminded about it.